modules/highlight/highlight.go

```diff
@@ -97,6 +86,22 @@ func Code(fileName, code string) string {
 		}
 		cache.Add(fileName, lexer)
 	}
+	return CodeFromLexer(lexer, code)
+}
+
+// CodeFromLexer returns a HTML version of code string with chroma syntax highlighting classes
+func CodeFromLexer(lexer chroma.Lexer, code string) string {
+	formatter := html.New(html.WithClasses(true),
+		html.WithLineNumbers(false),
+		html.PreventSurroundingPre(true),
+	)
+	if formatter == nil {
```

Comment:
`formatter` can not be nil. `html.New` always returns a `&Formatter{}`

Reply:
hmm... that comes from #12047 but you're right it's unnecessary and looks to have always been unnecessary.